client/src/components/auth/LayoutMainPage.vue

```diff
@@ -110,64 +129,89 @@ html {
 .second-column-in-secondary {
     width: 100%;
 }
+.place-for-form-vcol {
+    margin-top: 15vh;
+}
+.logo-secondary-div {
+    margin-top: 1.7vh;
+}
+.logo-primary-div {
+    display: none;
+    margin-top: 1.7vh;
+}
+/*main visual break out*/
 @media screen and (max-width: 960px) {
     .primary-vcol {
         order: 0;
-        height: 80vmax;
+        height: 100vh;
     }
     .secondary-vcol {
-        height: 80vmax;
+        height: 100vh;
     }
     .envelop-image {
         display: none;
     }
+    .normal-ok-sign {
+        display: none;
+    }
     .first-column-in-secondary {
         display: none;
     }
     .office-worker-image {
-        margin-top: 10px;
+        margin-top: 5vmin;
         min-height: 50vmin;
     }
+    .logo-primary-div {
+        display: block !important;
```

Comment:
I'm not sure you need !important here, above you do display: none, and it works, why doesn't it here?